www-client/firefox/firefox-98.0.2.ebuild

```diff
@@ -725,11 +726,11 @@ src_configure() {
 	# For future keywording: This is currently (97.0) only supported on:
 	# amd64, arm, arm64 & x86.
 	# Might want to flip the logic around if Firefox is to support more arches.
-	if use ppc64; then
-		mozconfig_add_options_ac '' --disable-sandbox
-	else
-		mozconfig_add_options_ac '' --enable-sandbox
-	fi
+	#if use ppc64; then
```

Comment:
it's ugly, can we just use "use ppc64 || use riscv"?

Reply:
Yes it's better.